tests/responses/subscriptions_for_userid.json

```diff
@@ -0,0 +1,21 @@
+{
+  "count": 1,
+  "previous": null,
+  "results": [{
+    "status": "active",
+    "deprovision_date": "2016-11-17",
+    "term": "P1Y",
+    "user_id": "fake-user-id-abcdef1234567890abc",
+    "auto_renew": false,
+    "billing_date": "2016-11-02",
+    "properties": {
+      "partner_id": "WL_BOTTOMLINE"
+    },
+    "expiry_date": "2016-11-02",
+    "start_date": "2015-11-02",
+    "sku": "12345678",
+    "type": "wl_subscription_type",
+    "id": "fake-sub-id-abcdef1234567890abcd"
+  }],
+  "next": null
+}
```

Comment:
You should not write integration tests for your models. The unit tests for `Subscription.list` collaborate with `Yola.list_subscriptions`. You're replacing it with a mock to verify the interaction and stub the results. I think this is correct and you should keep doing that. I'm just suggesting moving this dict directly into the test case since the data is important to the behavior you're verifying.